client/boot/index.js

```diff
@@ -52,6 +53,9 @@ function init() {
 
 	debug( 'Starting Calypso. Let\'s do this.' );
 
+	// prune sync-handler records
+	syncHandler.pruneRecordsFrom();
```

Comment:
I suggested using the [ms](https://www.npmjs.com/package/ms) microlib before, but folks didn't seem to want to pull it in.